klippy/extras/idle_timeout.py

```diff
@@ -31,11 +31,11 @@ def __init__(self, config):
                                     desc=self.cmd_SET_IDLE_TIMEOUT_help)
         self.state = "Idle"
         self.last_print_start_systime = 0.
+        self.mcu_startup_unixtime = time.time() - self.reactor.monotonic()
     def get_status(self, eventtime):
-        printing_time = 0.
-        if self.state == "Printing":
-            printing_time = eventtime - self.last_print_start_systime
-        return { "state": self.state, "printing_time": printing_time }
+        return { "state": self.state, "mcu_uptime": eventtime,
```

Comment:
Unless I missed something, I see no reason why the `printing_time` should be removed (especially as this is most likely in use in Klipper macros/UIs)

Reply:
From what I can tell, that's what @KevinOConnor proposed [here](https://github.com/Klipper3d/klipper/pull/6010#issuecomment-1405285448).
It's also just not very useful in its current state (only returns a value when a print is already running) and it can easily be replaced by using the new vars.

Reply:
I might have jumped the gun there: I just checked both Fluidd and Mainsail and confirmed this is not used (plus I guess there are better places to get the printing time anyway).

Carry on... 🙂

Reply:
Actually: There's seems to be an issues because klipper itself is using `printing_time` in [display.cfg](https://github.com/Klipper3d/klipper/blob/master/klippy/extras/display/display.cfg).

Reply:
Good spot @30350n! Not sure what @KevinOConnor would prefer in this case, but seems to me easier to keep the legacy `printing_time` as not to break existing installs!

Reply:
@pedrolamas I just get rid of the remaining references now. I've been running that code on my Voron for the last few weeks for testing purposes.